### PR TITLE
add 4.11 z-stream upgrade jobs and adjust test frequency

### PR DESCRIPTION
**Pull request by jianlinliu**

### CODE REVIEW

**Comment by jianlinliu** [comment]
cc @jiajliu @shellyyang1989 and @liangxia to review.

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__amd64-nightly-4.11-upgrade-from-stable-4.11.yaml [hunk 1]
```diff
@@ -0,0 +1,590 @@
...
+      product: ocp
+      stream: nightly
+      version: "4.11"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: aws-c2s-ipi-disconnected-private-fips-p2-f360
```
Comment (liangxia) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__amd64-nightly-4.11-upgrade-from-stable-4.11.yaml:59]: f360, :+1:

**Comment by liangxia** [comment]
/pj-rehearse ack
/approve

Adding the others required tags, so after @jiajliu or @shellyyang1989 adds lgtm, the PR will auto merge.

**Pull request merged**